src/components/NavigationBar/NavigationBar.tsx

```diff
@@ -0,0 +1,40 @@
+import imgLogo from 'assets/logo.png';
+import { ReactComponent as Paw } from 'assets/paw-solid.svg';
+export const NavigationBar = () => {
+  return (
+    <header className="header">
+      <div className="header__logo">
+        <img className="header__img" src={imgLogo} alt="logo" />
+        <div className="header__title">Puppylog</div>
+      </div>
+      <nav>
+        <ul>
+          <li className="header__link">
+            <div>
+              <Paw className="header__icon" />
+              About Us
+            </div>
+          </li>
```

Comment:
try remove div,
add :hover styles only for <li>

Reply:
I remove this and change to material ui navbar